projects/showcase/src/assets/css/google_font_roboto.css

```diff
@@ -0,0 +1,189 @@
+/* cyrillic-ext */
+@font-face {
```

Comment:
does it download all of these extended character sets on page load? do we need them?